Sandbox/PongLayer.h

```diff
@@ -6,6 +6,8 @@
 #include "Pong/Paddle.h"
 #include "Pong/PaddleController.h"
 
+#include <future>
```

Comment:
This was from me screwing around and is not necessary so I can either remove it or you can, just let me know

Reply:
I have removed.